### ISSUE TITLE
Disable sd-event in release/v2.0 once #358 is merged and release/v2.0 rebased

### ISSUE THREAD

**Issue by sangelovic** [Kistler-Group/sdbus-cpp#360]
Beware, [`release/v2.0`](https://github.com/Kistler-Group/sdbus-cpp/tree/release/v2.0) contains 9cc74e9942a5 which isn't supported by either `basu` or ConsoleKit2 (sd-compat.c). `*SdEvent` API will have to be disabled on FreeBSD via `#ifndef SDBUS_basu` or similar.

_Originally posted by @jbeich in https://github.com/Kistler-Group/sdbus-cpp/issues/358#issuecomment-1721585743_

**Comment by hellow554** [comment]
Hey @sangelovic 

#358 is merged.

I was just about to open a new ticket upon the state of the `release/v2.0` branch.
We use that branch in our code, because I needed that eventloop integration.

Meanwhile you're developing (awesome) stuff on the master branch, which isn't backported to v2.0

Is there really any advantage of having a 2.0 branch?

**Comment by sangelovic** [comment]
Hi @hellow554. Thanks for heads up. I've just rebased the `release/v2.0` branch on top of `master`. (So make sure the update the commit hash if you're referencing any, since the history has changed.) Btw, the branch is meant as a long-living development branch for v2 features and changes, not yet officially as a release branch for production use. However, feel free to use it; you'll just have to update the commit hash from time to time when the `release/v2.0` branch is rebased. The plan is that 
 the version `v2.0` will be released officially once all currently planned work that has a breaking effect on the API/ABI/behavior has been finished on `release/v2.0` and the branch has been merged to `master`, becoming the official main development line.
